io/disk/multipath_test.py

```diff
@@ -72,6 +72,8 @@ def setUp(self):
         # Check if given multipath devices are present in system
         self.wwids = self.params.get('wwids', default='').split(',')
         system_wwids = multipath.get_multipath_wwids()
+        if self.wwids == ['']:
```

Comment:
@tfauck This is already done in line 84

Reply:
yep - I wanted to say if system_wwids == ... fixing that - thanks